Piece.py

```diff
@@ -31,6 +31,9 @@ def __init__(self, name, isBlack):
 		# Raise KeyError if `name` not in `imageDatabase`
 		self.image = Piece.imageDatabase[self.name][self.isBlack] # ♔ ♚ ♕ ♛ ♗ ♝ ♘ ♞ ♙ ♟ ♖ ♜
 		self.isAlive = True
+  
+	def setImage(self):
```

Comment:
change name of the method to getImage, then it should be fine

Reply:
Approved